lefthook.yml

```diff
@@ -0,0 +1,15 @@
+pre-commit:
+  parallel: true
+  commands:
+    frontend-lint:
+      glob: '*.{js,ts,tsx}'
+      run: |
+        yarn lint {staged_files}
+        yarn prettier --write {staged_files}
+      stage_fixed: true
+    backend-format:
```

Comment:
I don't think we need backend-format in this repo

Reply:
Ah, true, copypasta strikes again. Removed it